packages/office-ui-fabric-react/src/utilities/selectableOption/SelectableDroppableText.types.ts

```diff
@@ -101,4 +101,10 @@ export interface ISelectableDroppableTextProps<TComponent, TListenerElement = TC
    * Input placeholder text. Displayed until option is selected.
    */
   placeholder?: string;
+
+  /**
+   * @default false

```

Comment:
Not sure if it matters for either intellisense or API extractor, but the default typically goes below the description